bin/verify-prod.js

```diff
@@ -0,0 +1,14 @@
+var base = require('taskcluster-base');
+var taskcluster = require('taskcluster-client');
+
+var cfg = base.config({profile: 'verify'});
+
+var purgeCache = new taskcluster.PurgeCache({
+  credentials: cfg.taskcluster.credentials
+});
+
+purgeCache.purgeCache(
+ 'verifyprovisioner',
+ 'verifyworker',
```

Comment:
We should use something with the words dummy... and standardize so we use the same for index testing... etc...